morceus-data/latin/stems/nominals/ls.nom

```diff
@@ -108797,6 +108863,10 @@ la_i^cus
 
 
 La_i+s		<itype>i^dis and i^dos</itype>	<gen>f.</gen>
+:le:Lais
+:no:La_	is_idis fem
+:no:La_	is_idos fem
```

Comment:
Same as above - can we or `orth` / `variant` this (or introduce a new term that is more appropriate)?

Reply:
added variant tag